<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoCreateShiftsForm</class>
 <widget class="QDialog" name="AutoCreateShiftsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schichten erstellen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Serie</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&amp;Wiederholung</string>
           </property>
           <property name="buddy">
            <cstring>comboBox_recurrence</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_recurrence">
           <item>
            <property name="text">
             <string>Täglich</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Wöchentlich</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Monatlich</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Jährlich</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Schicht &amp;Bezeichnung</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_shiftName</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_shiftName"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Beginn &amp;Datum/Zeit</string>
           </property>
           <property name="buddy">
            <cstring>dateTimeEdit_shift</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDateTimeEdit" name="dateTimeEdit_shift">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>&amp;Anzahl der Wiederholungen</string>
           </property>
           <property name="buddy">
            <cstring>spinBox_recurrenceCount</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBox_recurrenceCount"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Import</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>&amp;Typ</string>
           </property>
           <property name="buddy">
            <cstring>radioButton_iCal</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="radioButton_iCal">
             <property name="text">
              <string>iCal</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_RSS">
             <property name="text">
              <string>RSS</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>&amp;URL</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_URL</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_URL"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>&amp;Ignoriere Schichten vor</string>
           </property>
           <property name="buddy">
            <cstring>dateTimeEdit_ignoreBefore</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDateTimeEdit" name="dateTimeEdit_ignoreBefore">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_createShifts">
       <property name="text">
        <string>&amp;erstellen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox_recurrence</tabstop>
  <tabstop>spinBox_recurrenceCount</tabstop>
  <tabstop>lineEdit_shiftName</tabstop>
  <tabstop>dateTimeEdit_shift</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
